client/src/components/Ticketing/ticketingmanager/showings/ShowingUpdated/EventGeneralForm.tsx

```diff
@@ -0,0 +1,161 @@
+              name='eventdescription'
+              component={InputControl}
+              type='textarea'
+              label='Event Description'
+              id={0}
+              className={{
+                labelClass: 'text-sm font-semibold',
+                inputClass:
+                  'text-sm min-[450px]:text-md w-full rounded-lg p-1 border border-zinc-400',
+                inputGroupClass: 'flex flex-col',
+                controlClass: 'flex flex-col mb-2 text-zinc-800',
+              }}
+            />
+            <Field
+              name={'imageurl'}
+              component={InputControl}
+              label={'Image URL'}
+              type={'text'}
+              id={0}
+              disabled={values.imageurl === 'Default Event Image'}
+              className={{
+                labelClass: 'text-sm font-semibold',
+                inputClass:
+                  'text-sm min-[450px]:text-md w-full rounded-lg p-1 border border-zinc-400 disabled:bg-zinc-200 disabled:text-zinc-200',
+                inputGroupClass: 'flex flex-col',
+                controlClass: 'flex flex-col mb-2 text-zinc-800',
+              }}
+            />
+            <div className={'grid grid-cols-12 mb-2'}>
+              <div className={'flex flex-col justify-evenly col-span-6'}>
+                <label
+                  htmlFor={'defaultImageUrl'}
+                  className={
+                    'text-sm text-zinc-800 font-semibold text-center pb-1'
+                  }
+                >
+                  Use Default Image
+                </label>
+                <input
+                  name={'defaultImageUrl'}
+                  id={'defaultImageUrl'}
+                  type='checkbox'
+                  value={'default'}
+                  checked={disabledURL}
+                  onChange={async () => {
+                    await setFieldValue(
+                      'imageurl',
+                      !disabledURL ? 'Default Event Image' : '',
+                    );
+                    setDisabledURL(!disabledURL);
+                  }}
+                />
+              </div>
+              <div className={'flex flex-col justify-evenly col-span-6'}>
+                <label
+                  htmlFor={'active'}
+                  className={
+                    'text-sm text-zinc-800 font-semibold text-center pb-1'
+                  }
+                >
+                  Active
+                </label>
+                <input
+                  name={'active'}
+                  id={'active'}
+                  type='checkbox'
+                  value={values.active}
+                  checked={values.active}
+                  onChange={async () => {
+                    await setFieldValue('active', !values.active);
+                  }}
+                />
+              </div>
+            </div>
+          </div>
+          {/* Need to fix image sizing so height does not grow past needs of sibling div */}
```

Comment:
Can you create a Jira ticket for this?

Reply:
It is fixed, I just missed removing the comment.